tensorflow/core/kernels/cwise_op_gpu_select.cu.cc

```diff
@@ -33,6 +33,20 @@ struct SelectFunctor<GPUDevice, T> {
 };
 
 template <typename T>
+struct SelectScalarFunctor<GPUDevice, T> {
+  void operator()(const GPUDevice& d, typename TTypes<T>::Flat out,
+                  typename TTypes<bool>::ConstScalar cond,
+                  typename TTypes<T>::ConstFlat then_flat,
+                  typename TTypes<T>::ConstFlat else_flat) {
+    if (cond()){
```

Comment:
I think the whole point of this op was to have the 'cond' be on GPU though.  Otherwise you might as well just use tf.cond()